recipes-dom0/dom0-image-weston/files/meta-xt-extra/recipes-extended/guest-addons/guest-addons.bb

```diff
@@ -0,0 +1,33 @@
+# Copyright
+# License:
+#
+# Filename: guest-addons.bb
```

Comment:
This is Renesas IOMMU tests, right? Why to not name it as `renesas-ipmmu-tests` or something like this?
Because for me `guest-addons` is very generic and I expect to see some generic things inside.